pg_search/src/lib.rs

```diff
@@ -97,100 +83,6 @@ pub unsafe extern "C" fn _PG_init() {
     pgrx::hooks::register_hook(&mut TRACE_HOOK);
 }
 
```

Comment:
We've got some code here that requires `shared_preload_libraries` that we can also get rid of

Reply:
I think we need shared_preload_libraries as long as we still have a telemetry background worker.